cogs/ahk/ahk.py

```diff
@@ -919,6 +919,21 @@ async def handle_emoji_suggestion_reaction(self, reaction: disnake.RawReactionAc
 
                     return
 
+    @commands.Cog.listener()
+    async def on_member_remove(self, member: disnake.Member):
+        forum: disnake.ForumChannel = self.bot.get_channel(HELP_FORUM_CHAN_ID)
+
+        for thread in forum.threads:
+            if thread.owner_id != member.id:
+                continue
+            
+            if thread.archived:
+                continue
+            
+            if thread.locked:
+                continue
+            
+            await thread.send("❕ The thread owner is no longer in this server.")
```

Comment:
Would be nicer to put this in an embed, makes it very obvious that it's an actual message from the bot and not, for example, the output of a tag. Embed doesn't have to be fancy, an embed with a description is more than enough here

`await thread.send(embed=disnake.Embed(description="..."))`